tests/kernel/pending/src/pend.c

```diff
@@ -271,9 +270,12 @@ void task_low(void)
 	lifo_tests(FOURTH_SECOND, &task_low_state, my_lifo_get, k_sem_take);
 }
 
-void task_monitor(void)
+void test_pending(void)
 {
-	int result = TC_FAIL;
+	int old_prio = k_thread_priority_get(k_current_get());
```

Comment:
no need to save old_prio as we are not using anywhere.
also add a comment on why we are changing the priority just for sake of readability.

Reply:
Done